src/duk_debugger.h

```diff
@@ -41,6 +42,7 @@
 #define DUK_DBG_CMD_DETACH        0x1f
 #define DUK_DBG_CMD_DUMPHEAP      0x20
 #define DUK_DBG_CMD_GETBYTECODE   0x21
+#define DUK_DBG_CMD_APPREQUEST    0x23
```

Comment:
Will do.  Might be worth reordering things in the next incompatible break so that AppRequest/Notify are either the first or last command, it triggers my OCD seeing it in the middle of a list. :)